Gemfile

```
@@ -1,13 +1,13 @@
 source "https://rubygems.org"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.2.1"
+gem "rails"
```

```suggestion
gem "rails", "8.0.1"
```

Just to flag that per our conventions: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#gemfile-organisation the Rails gem should have a pin